Specs/Scene/Model/MetadataPipelineStageSpec.js

```diff
@@ -443,6 +446,187 @@ describe(
         );
       });
     });
+
+    it("Handles a tileset with metadata statistics", function () {
+      const modelPos = Cartesian3.fromDegrees(-75.152325, 39.94704);
+
+      const offset = new HeadingPitchRange(0, 0, 5.0);
+      scene.camera.lookAt(modelPos, offset);
+      //scene.camera.moveRight(5);
+      //scene.camera.moveUp(3);
+
+      const tilesetOptions = {
+        modelMatrix: Transforms.eastNorthUpToFixedFrame(modelPos),
+      };
+      return Cesium3DTilesTester.loadTileset(
+        scene,
+        tilesetWithMetadataStatistics,
+        tilesetOptions
+      ).then(function (tileset) {
+        expect(tileset).toBeDefined();
+        expect(tileset.tilesLoaded).toBe(true);
+
+        const metadataExtension = tileset.metadataExtension;
+        expect(metadataExtension).toBeDefined();
+        expect(metadataExtension.statistics).toBeDefined();
+
+        const root = tileset.root;
+        expect(root.hasEmptyContent).toBe(true);
+        expect(root.contentReady).toBe(true);
+
+        const child = root.children[0];
+        expect(child).toBeDefined();
+        expect(child.hasEmptyContent).toBe(false);
+        expect(child.contentReady).toBe(false);
+        expect(child.contentUnloaded).toBe(true);
+
+        const child1 = root.children[1];
+        expect(child1).toBeDefined();
+        expect(child1.hasEmptyContent).toBe(false);
+        expect(child1.contentReady).toBe(true);
+        expect(child1.contentUnloaded).toBe(false);
+
+        const content = child1.content;
+        expect(content).toBeDefined();
+
+        const model = content._model;
+        expect(model).toBeDefined();
```

Comment:
I had a hard time getting anything to show up in the camera. Here was my workflow:

1. Set up a Sandcastle to center the tileset in the Viewer canvas, with both tiles visible
2. Copy the camera parameters into the spec, and test. 
(Both tiles had empty content.)
3. Adjust the Sandcastle to zoom on the lower left corner of the _first_ child tile. 
(This time, the _second_ child tile was visible to the spec camera.)

Hence the extra tests to confirm where we had data.

Since these specs are really testing `createScene` and `Cesium3DTilesTester` rather than `MetadataPipelineStage`, I removed them from this spec. I'm happy to discuss further about how we can make spec camera setup more intuitive.